src/tests/Common/dirs.proj

```diff
@@ -30,17 +30,9 @@
     </ItemGroup>
 
     <ItemGroup Condition="'$(TestBuildMode)' == 'nativeaot' and '$(TargetOS)' == 'wasi'">
-      <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\PInvoke\*.csproj" />
-      <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\Preinitialization\*.csproj" />
-      <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\MultiModule\*.csproj" />
       <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\DynamicGenerics\*.csproj" />
-      <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\Exceptions\*.csproj" />
-      <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\UseSystemResourceKeys\*.csproj" />
-      <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\FrameworkStrings\*.csproj" />
       <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\Reflection\*.csproj" />
-      <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\TrimmingBehaviors\*.csproj" />
       <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\UnitTests\*.csproj" />
-      <DisabledProjects Include="$(TestRoot)nativeaot\SmokeTests\HelloWasm\ExceptionHandlingTests.Cpp.csproj" />
     </ItemGroup>
```

Comment:
The remaining three tests are failing on some seemingly unrelated issue with the interface dispatch always on stack. I wonder what will it turn out to be, there is nothing WASI-specific about CID.

Reply:
Found out the issue, should have been more obvious in retrospect since it is mentioned in #2269 -- there is partial `munmap`ping going on in `PalVirtualAlloc` that is used to allocate the CID caches.